script8.js

```diff
@@ -0,0 +1,36 @@
+function change() {
+    window.location.href = 'https://www.google.com';
```

Comment:
Your solution works! And here is an alternative approach.
`window.location.replace('https://www.google.com');`